documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -4,6 +4,9 @@
 Release Notes
 #############
 
+6.3.14
+======
+* Fixed fdbbackup start command that automatically configures database with backup workers to only do so when using partitioned logs. `(PR #4863) <https://github.com/apple/foundationdb/pull/4863>`_
```

Comment:
The release notes have been updated on `release-6.3`, so you'll need to pull from upstream and adjust this to avoid conflicts. Also, once this is merged, two more PRs need to made against `release-7.0` and `master`

Reply:
Done.